packages/lesswrong/components/sunshineDashboard/ModeratorUserInfo/ModeratorActionItem.tsx

```diff
@@ -0,0 +1,160 @@
+import moment from 'moment';
+import React, { useState } from 'react';
+import { isLowAverageKarmaContent } from '../../../lib/collections/moderatorActions/helpers';
+import { LOW_AVERAGE_KARMA_COMMENT_ALERT, LOW_AVERAGE_KARMA_POST_ALERT, MODERATOR_ACTION_TYPES } from '../../../lib/collections/moderatorActions/schema';
+import { registerComponent, Components } from '../../../lib/vulcan-lib';
+import { sortBy } from 'underscore';
+import Input from '@material-ui/core/Input';
+import DoneIcon from '@material-ui/icons/Done'
+import ClearIcon from '@material-ui/icons/Clear'
+import { useUpdate } from '../../../lib/crud/withUpdate';
+import classNames from 'classnames';
+
+const styles = (theme: ThemeType): JssStyles => ({
+  root: {
+    display: "flex",
+    alignItems: "center",
+    '&:hover': {
+      '& $clearIcon': {
+        opacity: .5
+      }
+    }
+  },
+  icon: {
+    width: 12,
+    height: 12,
+    cursor: "pointer",
+    opacity: .5,
+    marginRight: 10,
+    '&:hover': {
+      opacity: 1
+    }
+  },
+  date: {
+    marginLeft: 10,
+    display: "flex",
+    alignItems: "center",
+    '& input': {
+      width: 40,
+      textAlign: "center"
+    }
+  },
+  clickToEdit: {
+    cursor: "pointer"
+  },
+  margin: {
+    marginRight: 10
+  },
+  clearIcon: {
+    opacity: .1
+  }
+});
+
+export const ModeratorActionItem = ({classes, user, moderatorAction, comments, posts }: {
+  classes: ClassesType,
+  user: SunshineUsersList,
+  moderatorAction: ModeratorActionDisplay,
+  comments: Array<CommentsListWithParentMetadata>|undefined,
+  posts: Array<SunshinePostsList>|undefined
+}) => {
+
+  const { MetaInfo, LWTooltip } = Components
+
+  const parseExistingEndsInDays = () => {
+    const endedAtDate = moment(moderatorAction.endedAt)
+    const today = moment(new Date())
+    return endedAtDate.diff(today, "days")
+  }
```

Comment:
If this doesn't get called outside of setting the initial value for the `endsAfterDays` state, could probably just calculate the value once instead of wrapping it in a function